ndb/tests/unit/test_async_.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+
+from google.cloud.ndb import _eventloop
+from google.cloud.ndb import async_
+
+
+@unittest.mock.patch("google.cloud.ndb.async_._eventloop.EventLoop")
+def test_async_context(EventLoop):
+    one = unittest.mock.Mock()
```

Comment:
Prefer: `Mock(spec=())` (or auto spec with a particular class) to avoid "interface leakage" (a term I just made up here on the spot). I.e. if we're explicit about what spec we want, then unexpected behavior is less likely to happen.

**UPDATE**: It seems `spec=("run",)` is what you want?

Reply:
Done.